build_and_run_tests.sh

```diff
@@ -0,0 +1,11 @@
+set -x 
+
+#build
+bazel build --cxxopt='-std=c++14' //src/main:receive_main
+bazel build --cxxopt='-std=c++14' //src/main:send_main
+
+#bazel test --cxxopt='-std=c++14' --linkopt="-pthread" --test_output=all //tests:GoogleTest
+bazel run --cxxopt='-std=c++14' //tests:GoogleTest
+
```

Comment:
Why don't you have the tests, and you should inform the user where will be the test logs or open it for the user.

Reply:
So i tried running with `--test_output=all `, but for some reason this causes it to not identify the file path correctly?

![Screenshot from 2022-04-05 11-04-14](https://user-images.githubusercontent.com/97879338/161707782-d1386376-2342-495c-b8f3-9b1cf1600c87.png)

(whereas if I remove `--test_output=all ` it runs fine)

The reason why I did `run` instead of `test` is because then the test output is shown in the terminal like so:

![Screenshot from 2022-04-05 11-06-10](https://user-images.githubusercontent.com/97879338/161708115-d63bde8d-800f-4247-badf-916084dd00f8.png)

Reply:
2 Suggestions for that:

- you copy the test log to a folder that you display the location for the user to find it.
- you use cat to display the the log.

Reply:
Ah ok, yeah I have slightly changed the build script so it now will feed the output into `test-log.txt`